modules/compiler/source/base/include/base/pass_pipelines.h

```diff
@@ -53,6 +53,9 @@ struct BasePassPipelineTuner {
   /// @brief Whether or not to generate code for degenerate sub groups.
   bool degenerate_sub_groups = false;
 
+  /// @brief Whether or not the Handle Barriers Pass is going to be run.
+  bool handling_barriers = false;
```

Comment:
Make the default true so any customer targets inherit this since it fits most of our targets and in the past most our customer targets.

Reply:
if i know how to make only the G1 target set it to false then i could do that.

Reply:
under examples/refsi/refsi_g1_wi/compiler/refsi_g1_wi/source/refsi_pass_machinery.cpp -  you can set the flag directly there - although check with @frasercrmck as it's all changed since I last looked.